### PR TITLE
Missing type specifier in example

### PR DESCRIPTION
**Pull request by eshmoylova**
Missing type specifier in the example for Removing modifiers.

Trying out if I can create a pull request from the source without a local copy.

### CODE REVIEW

File: chapters/inheritance.tex [hunk 1]
```diff
@@ -1270,4 +1270,4 @@ \section{Selective Model Extension}\label{selective-model-extension}
 \end{lstlisting}
 In this case the deselection removes all of the connect-statements.
 \end{example}
-\end{nonnormative}
\ No newline at end of file
+\end{nonnormative}
```
Comment (eshmoylova) [inline chapters/inheritance.tex:1273]: I used "Edit this file" on github to modify it, and it also modified the white space at the end of the file. Not sure if this is crucial or how to revert that.

Comment (henrikt-ma) [reply chapters/inheritance.tex:1273]: It should be there.  Let's try this:
```suggestion
\end{nonnormative}

```

**Review by henrikt-ma — CHANGES_REQUESTED** [review]
Please commit the suggestion to restore the newline at end of file.

**Review by henrikt-ma — APPROVED** [review]
After correcting my bad attempt at cleaning up the changes, this is now back to where it started, and it is good.

**Review by HansOlsson — APPROVED** [review]
Ok.

**Pull request merged**